### PR TITLE
Spectral Resolution Equivalency.

### PR DESCRIPTION
**Pull request by SharonGoliath**
### Description

This pull request is to address ...

Fixes #<Issue Number>

### CODE REVIEW

**Comment by pllim** [comment]
Hi, @SharonGoliath ! Can you please provide some background and motivation for this pull request?

**Comment by SharonGoliath** [comment]
> Hi, @SharonGoliath ! Can you please provide some background and motivation for this pull request?

I was looking for a common implementation of the conversion of delta-wavelength / wavelength = delta-frequency / frequency, which is not yet available in astropy. It was suggested to me that creating the common implementation would be useful.

File: CHANGES.rst [hunk 1]
```diff
@@ -80,6 +80,7 @@ astropy.uncertainty
 astropy.units
 ^^^^^^^^^^^^^
 
+- Added spectral_resolution conversion to equivalencies.
```
Comment (pllim) [inline CHANGES.rst]:
```suggestion
- Added ``spectral_resolution`` conversion to equivalencies. [#9840]
```

**Comment by bsipocz** [comment]
@nmearl @astrofrog @mhvk - could you please comment on this and either review or remilestone or close.

**Comment by mhvk** [comment]
Apologies that this completely slipped under the radar. I removed the milestone for now as I feel this needs a bit of thought, which I think we're very short of right before the feature freeze deadline.

For later, I think we need to consider the balance of the usefulness of something quite specific and easy (but also easy to get wrong) with keeping things discoverable - the list of equivalencies is already very long. I also think, if we go for it, it should be extended to other spectral units (such as energy).

**Pull request closed without merging**